modules/govuk_crawler/templates/govuk_sync_mirror.erb

```diff
@@ -37,8 +37,9 @@ function get_error_page() {
   # Grab the 503 error page
   log "user.info" "Grabbing the latest error page"
 
-  test -d ${MIRROR_ROOT}/error || mkdir ${MIRROR_ROOT}/error/
-  curl -sf https://${STATIC_HOSTNAME}/templates/503.html.erb -o ${MIRROR_ROOT}/error/503.html
+  website_mirror_dir=${MIRROR_ROOT}/www.*
+  test -d ${website_mirror_dir}/error || mkdir ${website_mirror_dir}/error/
```

Comment:
```suggestion
  mkdir -p ${website_mirror_dir}/error/
```

Reply:
Yeah, I had the same thought. I'll do that, and do a bit of quoting of these variables. I'm not going to go shellchecking the whole file (although it's tempting).

Reply:
Oh yeah definitely don't feel obliged to polish the whole tu^H^Hthing 😂 Was only suggesting for readability cos you were changing that line anyway 👍 Thanks! ❤️